packages/server/lib/errors.coffee

```diff
@@ -345,6 +345,20 @@ API = {
         We looked but did not find a #{chalk.blue('cypress.json')} file in this folder: #{chalk.blue(arg1)}
         """
 
+      # TODO once we have a doc on global vs local install, add link
+      # QUESTION should we show different ways to run Cypress CLI depending on OS?
+      when "OPENED_CYPRESS_DIRECTLY"
+        """
+        It looks like you are running the Cypress binary directly.
+
+        This should only be done for DEVELOPMENT.
```

Comment:
I don't really understand what this means by DEVELOPMENT. Do you mean, like, internally developing in Cypress? Because reading this as a user, I think....aren't I developing?

Reply:
gotcha, do you want to change the warning message to something else?

Reply:
yes, that is the meaning - like we can internally run Cypress binary directly when working in this monorepo. Users when using Cypress should always go through the NPM package. Maybe something like this.

```
This is NOT the recommended way to run it. Please run ...
```

Reply:
I updated the message to the text @brian-mann wrote

Reply:
👍